src/api/config/feature.yml

```diff
@@ -0,0 +1,2 @@
+features:
+  image_templates: true
```

Comment:
Yes I'm talking about our RPM packages. If we have to ship it, we need to make sure that it's handled as config file so people can change it without things being overwritten. And then we need to handle the update case, like new entries needed in the file.

What happens if I have this file and I don't have `image_templates` in the settings?